plugins/new-relic/retention.yaml

```
@@ -400,37 +400,29 @@ presetScripts:
             'k8s.cluster.name': df.cluster_name,
             'px.cluster.id': df.cluster_id,
             'instrumentation.provider': df.pixie,
+            'db.system': df.db_system,
           },
           data=[
-              px.otel.metric.Summary(
-                  name='postgres.latency',
-                  count=df.latency_count,
-                  sum=df.latency_sum,
-                  unit='ms',
-                  quantile_values={
-                      0.0: df.latency_min,
-                      1.0: df.latency_max,
-                  },
-                  attributes={
-                      'db.system': df.db_system,
-                  },
-              ),
-              px.otel.metric.Gauge(
-              name='postgres.req_bytes',
-              description='',
-              value=df.req_bytes,
-              attributes={
-                'db.system': df.db_system,
-              },
-              ),
-              px.otel.metric.Gauge(
-              name='postgres.resp_bytes',
-              description='',
-              value=df.resp_bytes,
-              attributes={
-                'db.system': df.db_system,
+            px.otel.metric.Summary(
+              name='postgres.latency',
+              count=df.latency_count,
+              sum=df.latency_sum,
+              unit='ms',
+              quantile_values={
+                0.0: df.latency_min,
+                1.0: df.latency_max,
               },
-              )
+            ),
+            px.otel.metric.Gauge(
+            name='postgres.req_bytes',
+            description='',
+            value=df.req_bytes,
+            ),
+            px.otel.metric.Gauge(
+            name='postgres.resp_bytes',
+            description='',
+            value=df.resp_bytes,
+            )
```

indentation of this is wack still.
```suggestion
                name='postgres.req_bytes',
                description='',
                value=df.req_bytes,
            ),
            px.otel.metric.Gauge(
                name='postgres.resp_bytes',
                description='',
                value=df.resp_bytes,
            )
```